src/Aggregate/AggregateDescription.php

```diff
@@ -80,6 +83,57 @@ public static function describe(EventEngine $eventEngine): void
                 ->storeEventsIn(EventEngineUtil::fromAggregateNameToStreamName($aggregateName))
                 ->storeStateIn(EventEngineUtil::fromAggregateNameToDocumentStoreName($aggregateName));
         }
+
+        foreach (static::projectorsAggregateMapping() as $projectorClass) {
+            $eventsForProjector = $projectorClass::getEvents();
+
+            /** @var array<class-string> $aggregateRootClasses */
+            $aggregateRootClasses = [];
+
+            foreach ($eventsForProjector as $eventForProjector) {
+                $aggregateFromEvent = self::getAggregateFromEvent($eventForProjector);
+                if (! is_string($aggregateFromEvent)) {
+                    continue;
+                }
+
+                $aggregateRootClasses[] = $aggregateFromEvent;
+            }
+
+            $streams = [];
+            foreach ($aggregateRootClasses as $aggregateRootClass) {
+                $aggregateName = EventEngineUtil::fromAggregateClassToAggregateName($aggregateRootClass);
+                $aggregateStreamName = EventEngineUtil::fromAggregateNameToStreamName($aggregateName);
+
+                $streams[] = Stream::ofLocalProjection($aggregateStreamName);
+            }
+
+            $eventEngine->watch(...$streams)
+                ->with($projectorClass::getProjectionName(), $projectorClass)
+                ->filterEvents($eventsForProjector);
+        }
+    }
+
+    /**
+     * @param class-string $eventClass
+     *
+     * @return class-string|null
+     */
+    private static function getAggregateFromEvent(string $eventClass): ?string
+    {
+        $commandEventMappings = static::commandEventMapping();
+        $commandAggregateMappings = static::commandAggregateMapping();
+
+        foreach ($commandEventMappings as $command => $eventClasses) {
+            if (is_string($eventClasses)) {
+                $eventClasses = [$eventClasses];
+            }
+
+            if (array_key_exists($command, $commandAggregateMappings) && in_array($eventClass, $eventClasses, true)) {
+                return $commandAggregateMappings[$command];
+            }
+        }
+
+        return null;
```

Comment:
I would throw an exception here because if you return null, there is something wrong with your configuration.

Reply:
Done